app/src/main/java/com/ataulm/whatsnext/WhatsNextService.kt

```diff
@@ -19,6 +17,10 @@ internal class WhatsNextService(
         Token(authTokenApiResponse.accessToken, authTokenApiResponse.refreshToken)
     }
 
+    fun popularFilmsThisWeek(): PagingSource<String, FilmSummary> {
```

Comment:
PagingSource is a kotlin only class from paging-common. If we want to extract out modules, it's possible. See https://github.com/ataulm/art-collector/pull/34/